.travis.yml

```diff
@@ -1,34 +1,22 @@
 dist: trusty
 language: node_js
 node_js:
```

Comment:
is this node_js version same as the one in Docker image that @kmehant created?

Reply:
@JaDogg I think we don't require either python or nodejs on Travis as we are currently testing and building Bassa through docker containers.